tests/unit/Codeception/Module/MarkupValidatorTest.php

```diff
@@ -0,0 +1,402 @@
+<?php
+
+use Codeception\Module\MarkupValidator;
+
+class MarkupValidatorTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var MarkupValidator
+     */
+    protected $module;
+
+    protected function setUp()
+    {
+        $this->module = $this
+            ->getMockBuilder('\Codeception\Module\MarkupValidator')
+            ->setConstructorArgs(array(make_container()))
+            ->setMethods(array(
+                'getCurrentPageMarkup',
+            ))
+            ->getMock()
+        ;
+
+        $this->module->_initialize();
+    }
+
+    protected function tearDown()
+    {
+        $this->module->_cleanup();
+    }
+
+    public function testGetCurrentPageMarkupNoPhpBrowserNoWebDriver()
+    {
+        $this->module = $this
+            ->getMockBuilder('\Codeception\Module\MarkupValidator')
+            ->setConstructorArgs(array(make_container()))
+            ->enableProxyingToOriginalMethods()
+            ->getMock()
+        ;
+
+        $this->expectException('Exception');
```

Comment:
This test fails on PHP 5.4 with `PHP Fatal error:  Call to undefined method MarkupValidatorTest::expectException() in /home/travis/build/Codeception/Codeception/tests/unit/Codeception/Module/MarkupValidatorTest.php on line 40`

Codeception 2.2 supports PhpUnit 4.8, please use `setExpectedException` instead.